src/components/Button/native/Button.js

```diff
@@ -58,19 +43,19 @@ const Button = styled.TouchableOpacity`
       padding: ${p.theme.px([2, 3])};
     `,
   }[p.size])}
-  ${(p) => p.block ? 'width: 100%;' : ''}
-  ${(p) => p.disabled ? `
-    opacity: 0.5;
-  ` : ''}
+  ${(p) => p.block ? 'align-self: stretch;' : 'align-self: flex-start;'}
+  ${(p) => p.disabled ? 'opacity: 0.5;' : ''}
   `;
 
 Button.propTypes = {
+  ...TouchableOpacity.propTypes,
   color: PropTypes.string,
   kind: PropTypes.oneOf(['solid', 'outlined', 'transparent']),
-  shape: PropTypes.oneOf(['default', 'flat', 'rounded', 'circular']),
+  shape: PropTypes.oneOf(['default', 'flat', 'rounded']),
   size: PropTypes.oneOf(['small', 'regular', 'large']),
   block: PropTypes.bool,
   disabled: PropTypes.bool,
+  light: PropTypes.bool,
```

Comment:
What is light supposed to be? Can it not be formed as a combination of the rest of the props?